priv/rebar3_bsp_launcher.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh -eu
```

Comment:
We could consider making the language `rebar3_bsp` agnostic, by renaming both the script and the env variable and by taking the log file as input.